gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/dsl/graph/GraphTraversal.java

```diff
@@ -1852,6 +1853,17 @@ public default GraphTraversal<S, String> format(final String format) {
         return this.asAdmin().addStep(new FormatStep<>(this.asAdmin(), format));
     }
 
+    /**
+     * Parse the incoming traverser as a boolean value.
+     *
+     * @return the traversal with an appended {@link AsBoolStep}.
+     * @see <a href="http://tinkerpop.apache.org/docs/${project.version}/reference/#asBool-step" target="_blank">Reference Documentation - asBool Step</a>
```

Comment:
```suggestion
     * @see <a href="http://tinkerpop.apache.org/docs/${project.version}/reference/#asBool-step" target="_blank">Reference Documentation - asBool Step</a>
     * @since 3.8.0
```